easybuild/easyblocks/t/tensorflow.py

```diff
@@ -870,10 +870,6 @@ def test_step(self):
                 GPU_DEVICE: num_gpus_to_use,
             }
 
-        # These are used by the `parallel_gpu_execute` helper script from TF
```

Comment:
FYI: Moved that down after `if self._with_cuda:` was introduced

Reply:
:+1: